presto-iceberg/src/test/java/com/facebook/presto/iceberg/TestIcebergLogicalPlanner.java

```diff
@@ -77,6 +102,137 @@ protected QueryRunner createQueryRunner()
         return createIcebergQueryRunner(ImmutableMap.of("experimental.pushdown-subfields-enabled", "true"), ImmutableMap.of());
     }
 
+    @Test
+    public void testFilterPushdown()
+    {
+        Session pushdownFilterEnabled = pushdownFilterEnabled();
```

Comment:
The name pushdownFilterEnabled sounds like a boolean. Can we change it to sessionWithFilterPushdown or something similar?